src/Mono.Android/Java.Lang/JavaSystem.cs

```diff
@@ -0,0 +1,23 @@
+using System.Threading.Tasks;
+using System.Runtime.InteropServices;
+
+using Android.Runtime;
+
+namespace Java.Lang
+{
+	public sealed partial class JavaSystem : Java.Lang.Object
+	{
+		[DllImport (RuntimeConstants.InternalDllName, CallingConvention = CallingConvention.Cdecl)]
+		static extern void monodroid_javasystem_loadLibrary (string libname);
+
+		public static void LoadLibrary (string libname)
+		{
+			monodroid_javasystem_loadLibrary (libname);
+		}
+
+		public static Task LoadLibraryAsync (string libname)
```

Comment:
This should be wrapped in `#if ANDROID_34`, so that we don't inadvertently add new public API.

Reply:
`LoadLibraryAsync` is generated today, without any ifdefs, in `Java.Lang.JavaSystem.cs`:

```csharp
                // Metadata.xml XPath method reference: path="/api/package[@name='java.lang']/class[@name='System']/method[@name='loadLibrary' and count(parameter)=1 and parameter[1][@type='java.lang.String']]"
                [Register ("loadLibrary", "(Ljava/lang/String;)V", "")]
                public static unsafe void LoadLibrary (string libname)
                {
                        const string __id = "loadLibrary.(Ljava/lang/String;)V";
                        IntPtr native_libname = JNIEnv.NewString ((string?)libname);
                        try {
                                JniArgumentValue* __args = stackalloc JniArgumentValue [1];
                                __args [0] = new JniArgumentValue (native_libname);
                                _members.StaticMethods.InvokeVoidMethod (__id, __args);
                        } finally {
                                JNIEnv.DeleteLocalRef (native_libname);
                        }
                }

                public static global::System.Threading.Tasks.Task LoadLibraryAsync (string libname)
                {
                        return global::System.Threading.Tasks.Task.Run (() => LoadLibrary (libname!));
                }
```

Reply:
Doh, my bad!